src/nccl_ofi_rdma.c

```diff
@@ -3794,6 +3820,22 @@ static int accept(nccl_net_ofi_listen_comm_t *listen_comm,
 		}
 		l_comm->r_comm = r_comm;
 
+		/* Just in case ep changed, reset it here */
+		ep = (nccl_net_ofi_rdma_ep_t *)r_comm->base.base.ep;
```

Comment:
?

Reply:
The handling of this ep reference was messy. Next revision attempts to make it better.